### PR TITLE
Update/major version symfony3

### PR DESCRIPTION
**Pull request by brainwood**
**NOTE!! COMPOSER.JSON HAS DEPENDENCY ON CONCURRENT UPDATE BRANCH FOR TRANSACTOR WHICH NEEDS TO BE MERGED FIRST AND THIS COMPOSER.JSON CORRECTLY UPDATED TO POINT TO THE NEW VERSION**

Minor changes to make compatible with Symfony3

### CODE REVIEW

**Comment by tyler-sommer** [comment]
Going to leave this until we get the changes for `orkestra-transactor` merged, then we can deal with updating this bundle.

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "require": {
     "php": ">=5.3.3",
-    "orkestra/transactor": ">=1.0.0,<1.3.x-dev"
+    "orkestra/transactor": "dev-update/MajorVersion-Symfony3"
```
Comment (tyler-sommer) [inline composer.json]: So I think this should now point at `1.2.x-dev`. Should we cut `1.2.0` for `orkestra-transactor`? Then this project can require `>=1.2`.

I think either way, this bundle will need a major version bump

**Comment by brainwood** [comment]
Not following what you mean by cut 1.2.0 for orkestra-transactor?

**Comment by tyler-sommer** [comment]
Tag a new release. Currently, orkestra-transactor is on 1.1.5, dev-master is aliased as 1.2.x, so we could tag 1.2.0, change dev-master alias to 1.3.x, and update this bundle's requirement to `>=1.2`

**Comment by brainwood** [comment]
Okay so you are saying tag the current version as 1.2 then the new update
version when it gets merged in would be 1.3. The >=1.2 keeps it working
with both but the composer on the main project can specify 1.3 so that the
newer version is used?

That sounds fine to me

**Comment by tyler-sommer** [comment]
Alright so it appears that we won't need to bump major version. These changes are still compatible with 1.0 and 1.1 versions of orkestra-transactor.

**Pull request merged**